india_forecast_app/app.py

```diff
@@ -23,6 +24,13 @@
 version = india_forecast_app.__version__
 
 
+sentry_sdk.init(
+    dsn=os.getenv("SENTRY_DSN"),
+    environment=os.getenv("ENVIRONMENT", "local"),
+    traces_sampler=0.0 if os.getenv("ENVIRONMENT", "local") == "local" else 0.05
```

Comment:
If we want to catch every error, presuming that the code in `uk-pv-national-gsp-api` is actually working correctly, you might want that function from:

https://github.com/openclimatefix/uk-pv-national-gsp-api/blob/d969e955f9137c1ee55817820dbd9bbc71a9579b/src/utils.py#L123

unless it doesn't work how I think it does; but this basically just stops the tracing sending way too much data to Sentry, so checking whether it's an error and should always be sent makes sense to me.

Reply:
ok, ill change it to what it is in the API. I was trying to reduce the code, but perhaps it is worth putting it